SPECS/hdf5/hdf5.spec

```
@@ -0,0 +1,1054 @@
+    --enable-parallel \
+    --exec-prefix=%{_libdir}/$mpi \
+    --libdir=%{_libdir}/$mpi/lib \
+    --bindir=%{_libdir}/$mpi/bin \
+    --sbindir=%{_libdir}/$mpi/sbin \
+    --includedir=%{_includedir}/$mpi-%{_arch} \
+    --datarootdir=%{_libdir}/$mpi/share \
+    --mandir=%{_libdir}/$mpi/share/man \
+    --with-default-plugindir=%{_libdir}/$mpi/hdf5/plugin
+  sed -i -e 's! -shared ! -Wl,--as-needed\0!g' libtool
+  %make_build LDFLAGS="%{__global_ldflags} -fPIC -Wl,-z,now -Wl,--as-needed"
+  module purge
+  popd
+done
+
+
+%install
+%make_install -C build
+rm %{buildroot}%{_libdir}/*.la
+# Fortran modules
+mkdir -p %{buildroot}%{_fmoddir}
+mv %{buildroot}%{_includedir}/*.mod %{buildroot}%{_fmoddir}
+# Fix fortran module include dir https://bugzilla.redhat.com/show_bug.cgi?id=1971826
+sed -i -e 's,%{_includedir},%{_fmoddir},' %{buildroot}%{_bindir}/h5fc
+# Plugin directory
+mkdir -p %{buildroot}%{_libdir}/hdf5/plugin
+for mpi in %{?mpi_list}
+do
+  module load mpi/$mpi-%{_arch}
+  %make_install -C $mpi
+  rm %{buildroot}/%{_libdir}/$mpi/lib/*.la
+  # Fortran modules
+  mkdir -p %{buildroot}${MPI_FORTRAN_MOD_DIR}
+  mv %{buildroot}%{_includedir}/${mpi}-%{_arch}/*.mod %{buildroot}${MPI_FORTRAN_MOD_DIR}/
+  # Fix fortran module include dir https://bugzilla.redhat.com/show_bug.cgi?id=1971826
+  sed -i -e "s,%{_includedir},${MPI_FORTRAN_MOD_DIR}," %{buildroot}%{_libdir}/$mpi/bin/h5pfc
+  # Plugin directory
+  mkdir -p %{buildroot}%{_libdir}/$mpi/hdf5/plugin
+  module purge
+done
+#Fixup example permissions
+find %{buildroot}%{_datadir} \( -name '*.[ch]*' -o -name '*.f90' \) -exec chmod -x {} +
+
+#Fixup headers and scripts for multiarch
+%ifarch x86_64 ppc64 ia64 s390x sparc64 alpha
+sed -i -e s/H5pubconf.h/H5pubconf-64.h/ %{buildroot}%{_includedir}/H5public.h
+mv %{buildroot}%{_includedir}/H5pubconf.h \
+   %{buildroot}%{_includedir}/H5pubconf-64.h
+for x in h5c++ h5cc h5fc
+do
+  mv %{buildroot}%{_bindir}/${x} \
+     %{buildroot}%{_bindir}/${x}-64
+  install -m 0755 %SOURCE1 %{buildroot}%{_bindir}/${x}
+done
+%else
+sed -i -e s/H5pubconf.h/H5pubconf-32.h/ %{buildroot}%{_includedir}/H5public.h
+mv %{buildroot}%{_includedir}/H5pubconf.h \
+   %{buildroot}%{_includedir}/H5pubconf-32.h
+for x in h5c++ h5cc h5fc
+do
+  mv %{buildroot}%{_bindir}/${x} \
+     %{buildroot}%{_bindir}/${x}-32
+  install -m 0755 %SOURCE1 %{buildroot}%{_bindir}/${x}
+done
+%endif
+# rpm macro for version checking
+mkdir -p %{buildroot}%{macrosdir}
+cat > %{buildroot}%{macrosdir}/macros.hdf5 <<EOF
+# HDF5 version is
+%%_hdf5_version %{version}
+EOF
+
+
+%if %{with java}
+# Java
+mkdir -p %{buildroot}%{_libdir}/%{name}
+mv %{buildroot}%{_libdir}/libhdf5_java.so %{buildroot}%{_libdir}/%{name}/
+%endif
+
+%check
+%ifarch %{ix86} s390x
+# i386: java TestH5Arw segfaults
+# s390x: Testing inserting objects to create first direct block in recursive indirect blocks five levels deep*FAILED*
+make -C build check || :
+%else
+make -C build check
+%endif
```

We don't build for these architectures, let's remove this part of the spec.
```suggestion
make -C build check
```